collectors/bzimport/convertors.py

```diff
@@ -613,6 +613,7 @@ def get_meta_attr(self, cve_id):
         meta_attr["alias"] = self.alias
         meta_attr["depends_on"] = self.depends_on
         meta_attr["related_cves"] = [c for c in self.cve_ids if c != cve_id]
+        meta_attr["summary"] = self.flaw_bug["summary"]
```

Comment:
I would call this eg. `bz_summary` (or somehow differently) because the term summary in OSIDB actually means Bugzilla Doc Text. WDYT?

Reply:
Makes sense, done!